spec/ui/ui_spec.rb

```diff
@@ -24,81 +28,119 @@
     expect(ui.ask_for_another).to eq(false)
   end
 
+  it "prints the prompt message for repeating search" do
+    input.string = "\n"
+    ui.ask_search_again
+    expect_to_print(UI::SEARCH_AGAIN)
+  end
+
+  it "reads a positive answer to search again" do
+    input.string = "y"
+    expect(ui.ask_search_again).to eq(true)
+  end
+
+  it "reads a negative answer to search again" do
+    input.string = "n"
+    expect(ui.ask_search_again).to eq(false)
+  end
+
+  it "prints the prompt message for expanding a contact's details" do
+    input.string = "\n"
+    ui.ask_to_expand
+    expect_to_print(UI::EXPAND)
+  end
+
+  it "reads a positive answer to expand contact details" do
+    input.string = "y"
+    expect(ui.ask_to_expand).to eq(true)
+  end
+
+  it "reads a negative answer to expand contact details" do
+    input.string = "n"
+    expect(ui.ask_to_expand).to eq(false)
+  end
+
+  it "prints the prompt message for expanding a contact's details" do
+    input.string = "\n"
+    ui.ask_another_field
+    expect_to_print(UI::ANOTHER_FIELD)
+  end
+
+  it "reads a positive answer to ask another field" do
+    input.string = "y"
```

Comment:
How many lines of code would need to change if a requirement change came in that makes in mandatory to actually type `yes` or `no` instead of `y` and `n`?